index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Portfolio</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="./style.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    </head>
+    <body>
+        <section class="appbar">
+            <div class = "toolbar">
+                <div class="mylogo">
+                    My Logo
+                </div>
+                <nav class="menu-container">
+                    <a href="" class="hamburger" ">
+                        <i class="fa fa-bars"></i>
```

Comment:
- [ ] Kindly change your `hamburger` to a button element instead, it does not have to be a link element and redirects the user to the homepage, instead, we will use it for displaying the mobile menu in future project implementations, as for now, it has to be a simple button.

- [ ] KIndly remove this extra mark `"`, it seems inappropriate.
```suggestion
                    <a href="" class="hamburger">
                        <i class="fa fa-bars"></i>
```